docs/guides/lint-asyncapi.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Lint AsyncAPI
+---
+
+# Lint AsyncAPI with Redocly CLI
+
+In addition to providing lint functionality for multiple OpenAPI formats, Redocly CLI also has support for AsyncAPI.
+Redocly CLI supports the following linting approaches with AsyncAPI documents:
+
+* AsyncAPI document validation, including full binding validation for [supported protocols](#supported-protocols).
+* Supported versions:
+  - AsyncAPI 2.6
+  - earlier versions in the 2.x family may also validate successfully
+* Built-in rules for checking common standards requirements (see the [list of AsyncAPI rules](#asyncapi-rules)).
+* [Configurable rules](../rules/configurable-rules.md) so that you can build your own rules following common patterns
+* [Custom plugins](../custom-plugins/index.md) for advanced users that need additional functionality
+
+## Lint an existing AsyncAPI file
+
+Redocly CLI takes its settings from a `redocly.yaml` configuration file. Below
+is an example of a simple configuration file for validating an AsyncAPI file is
+in the expected format:
+
+```yaml
+extends: []
+
+rules:
+  spec: error
+```
+
+The empty `extends` element instructs Redocly CLI not to use any existing
+rulesets, but to emit an error if the `spec` rule finds any problem. This rule
+checks that the document structure matches what is expected by the AsyncAPI
+specification.
+
+With this configuration file, and your AsyncAPI description file, run the linting command:
+
+```
+redocly lint asyncapi.yaml
+```
+
+The output describes any structural problems with the document, or reports that it is valid.
```

Comment:
A distinction between warnings and errors would be useful here.

"The default linting output includes both errors and warnings. Errors are structural problems that invalidate your AsyncAPI document. Warnings are typically problems that impact quality. The linter will report that your spec is valid if there are no errors (even if there are warnings)."

Reply:
This should exist somewhere, but not in a specific guide. If we don't have it, let's add it!